Frontend/ui-driver/src/Screens/Types.purs

```diff
@@ -788,7 +788,8 @@ type HomeScreenData =  {
   config :: AppConfig,
   triggerPatchCounter :: Int,
   peekHeight :: Int,
-  driverGotoState :: DriverGoToState
+  driverGotoState :: DriverGoToState,
+  snappedOrigin :: Maybe Location
```

Comment:
what's snappedOrigin for?

Reply:
This is the source (on route) which user is sending, And we are getting it from `"/" <> rideId <>"/route"` API